satip.c

```diff
@@ -87,7 +87,10 @@ const char *cPluginSatip::CommandLineHelp(void)
          "                                define hard-coded SAT>IP server(s)\n"
          "  -D, --detach                  set the detached mode on\n"
          "  -S, --single                  set the single model server mode on\n"
-         "  -n, --noquirks                disable all the server quirks\n";
+         "  -n, --noquirks                disable all the server quirks\n"
+         "  -r, --portrange=<start>       set a range of ports used for the rt[c]p server\n"
```

Comment:
This makes only sense if the range is defined in the ephemeral portrange. Most unix systems are using  32768 to 61000. If no portrange is defined via the cmdline parameter, the plugin binds to ephemeral ports. I would rather specify a port far below that, e.g starting at 4000. These ports normally never binds "automatically".